src/Analysis/Engine/Test/AnalysisTest.cs

```diff
@@ -4403,7 +4403,7 @@ class oar(list):
                 var completions = await server.SendCompletion(uri, 2, 8);
 
                 completions.Should().HaveItem("append")
-                    .Which.Should().HaveInsertText("append(self, value):\r\n\treturn super(oar, self).append(value)");
+                    .Which.Should().HaveInsertText("append(self, value):\r\n    return super(oar, self).append(value)");
```

Comment:
How do we determine if it is a `\t` or a set of spaces? Is there a setting for it?

Reply:
Per `GetCompletionsForOverride` it always uses spaces. I believe per PEP spaces are strongly preferred.